vendor/k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver/log_healthz.go

```diff
@@ -0,0 +1,29 @@
+package openshiftkubeapiserver
+
+import (
+	"k8s.io/klog"
+	"net/http"
+	"strings"
+
+	"k8s.io/apiserver/pkg/server/healthz"
+)
+
+// LogRequestToHealthz logs a user-agent that uses /healthz endpoint
+var LogRequestToHealthz healthz.HealthChecker = &logRequestToHealthz{}
+
+type logRequestToHealthz struct {}
+
+func (l *logRequestToHealthz) Name() string {
+	return "logRequestToHealthz"
+}
+
+func (l *logRequestToHealthz) Check(r *http.Request) error {
+	userAgent := r.Header.Get("User-Agent")
+
+	// it seems that identifies itself us "kube-probe/1.18+",
+	if len(userAgent) > 0 && !strings.Contains(userAgent, "kube-probe") {
```

Comment:
we could also log when a `User-Agent` hasn't been provided

Reply:
does it happen?

Reply:
good question, we could find out by logging all requests